sanic/server.py

```diff
@@ -295,6 +309,29 @@ def data_received(self, data):
         except Exception:
             logger.exception("protocol.data_received")
 
+    # -------------------------------------------- #
+    # Python 3.7+
+    # -------------------------------------------- #
+
+    def get_buffer(self, sizehint=-1):
```

Comment:
what is `sizehint` used for here ?

Reply:
> sizehint is the recommended minimum size for the returned buffer. It is acceptable to return smaller or larger buffers than what sizehint suggests. When set to -1, the buffer size can be arbitrary. It is an error to return a buffer with a zero size.

[Source](https://docs.python.org/3/library/asyncio-protocol.html#asyncio.BufferedProtocol.get_buffer)

Reply:
We are ignoring it here.